### DISCUSSION TITLE
Forwardemail alias not receiving external emails

### DISCUSSION THREAD

**Discussion by jwclack52** [forwardemail/forwardemail.net#234] [General]
Everything is verified - my domain records and alias setup.  FE says everything is fine, but external emails aren't showing up.  Sent emails get sent and DO show up in my sent folder, so SMTP is working, but IMAP isn't.  Testing shows a 402 / authentication failed error.  IMAP/SMTP passwords are the same, aren't the?  Strange thing is that I can check for messages w/out error AND emails sent to the alias from the alias show up in my inbox - just no other messages from the outside.  I've reported to support, but they haven't responded for days.

**Comment by dannypernik** [comment]
@jwclack52 are you using Gmail with the "Treat as an alias" setting checked by chance? The instructions say to uncheck this box. I was hoping that things would work with this setting checked because otherwise, my own email annoyingly gets added to the Cc field when I click "reply all". Unfortunately, I just sent myself a test email, and it seems that I do not receive email with this setting checked. After unchecking the alias setting and sending another test, I received both emails – but I received the initial test after receiving the 2nd email I sent. I believe this means that the first email initially failed but was automatically retried and succeeded.

I'm new to ForwardEmail and have been disappointed with the absence of support. I actually started a discussion a few days ago asking if I could check the "alias" box and another question, but I have not received a reply.

**Reply by jwclack52** [reply]
Nope.  I'm using a domain email acct w/ Hostgator.

**Comment by titanism** [comment]
Hi folks and apologies for the support delays - we suffered a DDOS attack the previous week and have been catching up on requests.  @jwclack52 what client are you using?  are there any logs that provide a full error message for the 402 error?  IMAP/SMTP passwords are the same, correct - are you using a specific alias or a trying to use a wildcard, or something else?

**Reply by jwclack52** [reply]
Tbird. I used a website tool to check the status of IMAP wherein it showed me the error:  https://www.dotcom-tools.com/email-server-test

Right now I'm trying to access using a single alias, but your guides are confusing.  I was not prompted to add a record specific to the email address and your guide tells me I'm good, but obviously not.  I could use a little help w/ your "wildcard" records.

**Comment by jwclack52** [comment]
Okay - I think I'm giving up.  Email HOSTING doesn't seem to be working and support is ghosting me.  Time to ask for a refund.

**Comment by titanism** [comment]
Hi there 👋  We're not ignoring you - we informed you by email that you cannot do recursive forwarding (e.g. attempt to set a wildcard forwarding address to the same domain as the one you're configuring).  Please enable IMAP for your alias, delete any plaintext TXT records you added (not the encrypted one required for verification), and set up an email client such as Thunderbird.  We will try to review your communications again shortly and follow up if we see something else you might need to do.

**Reply by dannypernik** [reply]
@titanism are you ignoring me, then? No reply to multiple emails or this discussion item despite me upgrading to $9 a month.
https://github.com/orgs/forwardemail/discussions/232

**Comment by jwclack52** [comment]
Already tried that.  Had the same problem prior to adding the extra forward (it had no effect on the issue).

 

Please explain to me how your HOSTING service works.  What I want to do is send/receive emails for, say, ***@***.*** ***@***.***> , via your HOSTING servers.  Your setup web pages tell me that I’m verified for both IMAP and SMTP, but emails are STILL not being received by your HOSTING servers.

**Reply by jwclack52** [reply]
I think it's working - for the most part.  Thing is my alias can't receive emails from my Hostgator-hosted email account for another domain.  It receives messages from other email accounts.  I never get any bounce notifications either.

**Comment by jwclack52** [comment]
So, things are working with two exceptions:

- If I send an email from my Hostgator hosted email to my jim@... alias, it never shows up.  However, other email accounts successfully send emails that do show up.
- I'm trying to get a simple forwarding alias to work.  Trying to forward jwc@... to jim@...  doesn't seem to work.  What am I doing wrong?

**Comment by jwclack52** [comment]
C'mon, guys.  I really want this to work (and I think after initial learning curve it will), but I need some help getting started.  I've been asking about a couple of issues for a while now.

**Comment by titanism** [comment]
Hi @jwclack52  - I know we followed up by email as well - are you still having issues here?

**Comment by jwclack52** [comment]
Yes.  Same two issues.  You haven’t addressed either.

**Reply by jwclack52** [reply]
Yes - same two issues I've been telling you about for a month.

**Comment by titanism** [comment]
Hi there - thanks for your patience, there should be an email response in your inbox shortly.  Going to redact/delete the above comments to keep your information private.

Edit: Looks like you already redacted it, thank you!

**Comment by titanism** [comment]
@jwclack52 just to be clear - we also notified you by email about this, but you can't use two different MX services at the same time on the same domain.  Email will always go to the lowest priority MX server, in this case, your domain only has our records.  You can't also be trying to use Hostgator or Hostinger or whoever your other service is.  You can use us, and then use a mail client like Thunderbird, Apple Mail, Outlook, etc to connect to IMAP servers and send/receive mail (NOT using Hostgator).

**Reply by jwclack52** [reply]
NOT RESOLVED.  

I waited weeks for this response?  First of all, I had TWO issues and I'm not sure which one this even addresses.  PLEASE REREAD the message above.  When I try to send an email to jim@cumberlandbio.org from a **DIFFERENT EMAIL ACCOUNT FROM A DIFFERENT DOMAIN** (jim@clack.us, hosted by Hostgator), THEREFORE, DIFFERENT MX RECORD (MX / @ / mail.clack.us), IT NEVER ARRIVES.  I repeat:  DIFFERENT EMAIL ON DIFFERENT DOMAIN W/ DIFFERENT MX RECORD.  Got that?

The second issue involves, AS I WROTE ABOVE, creating a true alias, in this case jwc@cumberlandbio.org, that forwards to jim@cumberlandbio.org.  You even have a field in your form to do this:
**Forwarding Recipients (optional)
jim@cumberlandbio.org**
Recipients must be a line-break/space/comma separated list of valid email addresses, fully-qualified domain names ("FQDN"), IP addresses, and/or webhook URL's. We will automatically remove duplicate entries for you and perform validation when you click "Update Alias" below. We do not support forwarding recursively more than one level deep. If you forward recursively more than one level deep, then your desired configuration may not work as intended.

DOES NOT WORK.  EMAILS SENT TO JWC@CUMBERLANDBIO.ORG NEVER GET FORWARDED TO JIM@CUMBERLANDBIO.ORG.

Now, PLEASE, answer my questions with reasonable answers.

**Comment by jwclack52** [comment]
NOT RESOLVED:

 

NOT RESOLVED.  

 

I waited weeks for this response?  First of all, I had TWO issues and I'm not sure which one this even addresses.  PLEASE REREAD the message above.  When I try to send an email to ***@***.*** ***@***.***>  from a DIFFERENT EMAIL ACCOUNT FROM A DIFFERENT DOMAIN ***@***.*** ***@***.***> , hosted by Hostgator), THEREFORE, DIFFERENT MX RECORD (MX / @ / mail.clack.us), IT NEVER ARRIVES.  I repeat:  DIFFERENT EMAIL ON DIFFERENT DOMAIN W/ DIFFERENT MX RECORD.  Got that?

 

The second issue involves, AS I WROTE ABOVE, creating a true alias, in this case ***@***.*** ***@***.***> , that forwards to ***@***.*** ***@***.***> .  You even have a field in your form to do this:

**Forwarding Recipients (optional)

***@***.***** ***@***.*****> 

Recipients must be a line-break/space/comma separated list of valid email addresses, fully-qualified domain names ("FQDN"), IP addresses, and/or webhook URL's. We will automatically remove duplicate entries for you and perform validation when you click "Update Alias" below. We do not support forwarding recursively more than one level deep. If you forward recursively more than one level deep, then your desired configuration may not work as intended.

 

DOES NOT WORK.  EMAILS SENT TO ***@***.*** ***@***.***>  NEVER GET FORWARDED TO ***@***.*** ***@***.***> .

 

Now, PLEASE, answer my questions with reasonable answers.

**Comment by jwclack52** [comment]
NOT RESOLVED:



NOT RESOLVED.  

 

I waited weeks for this response?  First of all, I had TWO issues and I'm not sure which one this even addresses.  PLEASE REREAD the message above.  When I try to send an email to ***@***.*** ***@***.***>  from a DIFFERENT EMAIL ACCOUNT FROM A DIFFERENT DOMAIN ***@***.*** ***@***.***> , hosted by Hostgator), THEREFORE, DIFFERENT MX RECORD (MX / @ / mail.clack.us), IT NEVER ARRIVES.  I repeat:  DIFFERENT EMAIL ON DIFFERENT DOMAIN W/ DIFFERENT MX RECORD.  Got that?

 

The second issue involves, AS I WROTE ABOVE, creating a true alias, in this case ***@***.*** ***@***.***> , that forwards to ***@***.*** ***@***.***> .  You even have a field in your form to do this:

**Forwarding Recipients (optional)

***@***.***** ***@***.*****> 

Recipients must be a line-break/space/comma separated list of valid email addresses, fully-qualified domain names ("FQDN"), IP addresses, and/or webhook URL's. We will automatically remove duplicate entries for you and perform validation when you click "Update Alias" below. We do not support forwarding recursively more than one level deep. If you forward recursively more than one level deep, then your desired configuration may not work as intended.

 

DOES NOT WORK.  EMAILS SENT TO ***@***.*** ***@***.***>  NEVER GET FORWARDED TO ***@***.*** ***@***.***> .

 

Now, PLEASE, answer my questions with reasonable answers.

 

On April 9, 2024 3:26:26 PM titanism ***@***.***> wrote:

@jwclack52 <https://github.com/jwclack52>  just to be clear - we also notified you by email about this, but you can't use two different MX services at the same time on the same domain. Email will always go to the lowest priority MX server, in this case, your domain only has our records. You can't also be trying to use Hostgator or Hostinger or whoever your other service is. You can use us, and then use a mail client like Thunderbird, Apple Mail, Outlook, etc to connect to IMAP servers and send/receive mail (NOT using Hostgator).

**Comment by titanism** [comment]
Locking this conversation as we have an email thread.  Thank you.
